app/assets/javascripts/header.js

```diff
@@ -94,4 +68,36 @@ function setHeight() {
   $('#header_space').css({"height": the_height});
 }
 
+function expandButton(event) {
+  hideSmallTopNav();
+  console.log(event.data);
+  if (event.data.value % 2 == 0) {
+    showExpandMe();
+  } else {
+    hideExpandMe();
+  }
+  setHeight();
+  event.data.value++;
+}
+
+function headerMouseLeave() {
+  if ($('#expand_moment').length && $('#expand_moment')[0].classList.contains('display_block')) {
```

Comment:
Plus, now that there are tests, it's easy to know if something went wrong when extracting out a function here